recipes/binutils/binutils.inc

```diff
@@ -12,12 +12,12 @@ BUGTRACKER = "http://sourceware.org/bugzilla/"
 
 RECIPE_TYPES = "cross sdk-cross machine canadian-cross"
 
-inherit autotools make-vpath gettext
+inherit autotools make-vpath
 
 require conf/fetch/gnu.conf
 SRC_URI = "${GNU_MIRROR}/binutils/binutils-${PV}.tar.bz2 "
 
-DEPENDS = "native:flex native:bison"
+DEPENDS = "native:flex native:bison native:makeinfo native:m4"
```

Comment:
@diorcety where do you expect to get native:makeinfo from?  Did you send a PR with makeinfo recipe, or should it be added to ASSUME_PROVIDED?